src/base/bittorrent/session.h

```diff
@@ -402,6 +402,9 @@ namespace BitTorrent
         void setAnnounceIP(const QString &ip);
         int maxConcurrentHTTPAnnounces() const;
         void setMaxConcurrentHTTPAnnounces(int value);
+        bool isReannounceWhenAddressChanged() const;
+        void setReannounceWhenAddressChanged(bool enabled);
+        void reannounceToAllTrackers();
```

Comment:
I suppose this is a `const` function: `void reannounceToAllTrackers() const;`

Reply:
Added const qualifier to method